worldengine/draw.py

```diff
@@ -477,3 +554,8 @@ def draw_ancientmap_on_file(world, filename, resize_factor=1,
                     draw_biome, draw_rivers, draw_mountains, draw_outer_land_border, 
                     verbose)
     img.complete()
+
+def draw_scatter_plot_on_file(world, filename):
+    img = ImagePixelSetter(512, 512, filename)
+    draw_scatter_plot(world, 512, img)
```

Comment:
but draw_scatter_plot doesn't take just 2 parameters?
perhaps we could use constants to make it clear the meaning of the value 512. It is absolutely fine to have a chart with fixed size, it should be just clear that the first 512x512 means the chart is 512 pixels wide and tall